example/ios/example/Info.plist

```diff
@@ -53,14 +51,6 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-	    <string>instagram-stories</string>
-        <string>instagram</string>
-		<string>facebook</string>
-		<string>whatsapp</string>
-		<string>mailto</string>
-	</array>
```

Comment:
Are you sure you don't need these anymore?

Reply:
Actually I need it, thanks for pointing this. 😅

Reply:
Done!